spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/pulsar/PulsarProperties.java

```diff
@@ -801,6 +802,11 @@ public static class Listener {
 		 */
 		private SchemaType schemaType;
 
+		/**
+		 * Number of threads used by listener container.
+		 */
+		private Integer concurrency;
```

Comment:
Can you add this support to the Reactive side as well?

See https://github.com/spring-projects/spring-boot/blob/fa686bb593f2a3b85497546d3995c4050501ef61/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/pulsar/PulsarReactivePropertiesMapper.java#L92

Reply:
I've addressed this in the latest PR update.